Detectors/TPC/workflow/src/tpc-calibrator-dEdx.cxx

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "TPCWorkflow/CalibratordEdxSpec.h"
+#include "Framework/runDataProcessing.h"
+
+using namespace o2::framework;
+
+WorkflowSpec defineDataProcessing(ConfigContext const&)
+{
+  using namespace o2::tpc;
+  return WorkflowSpec{getCalibratordEdxSpec()};
```

Comment:
We should also have this `configKeyValues` stuff you put in `tpc-calib-dEdx.cxx` here, I think.